src/main/java/seedu/address/ui/BookListPanel.java

```diff
@@ -0,0 +1,49 @@
+package seedu.address.ui;
+
+import java.util.logging.Logger;
+
+import javafx.collections.ObservableList;
+import javafx.fxml.FXML;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
+import javafx.scene.layout.Region;
+import seedu.address.commons.core.LogsCenter;
+import seedu.address.model.book.Book;
+
+/**
+ * Panel containing the list of persons.
```

Comment:
Shouldn't this be list of books?

Reply:
Thank you, I will update the doc to list of books!